src/game/level/boxes.c

```diff
@@ -33,7 +36,11 @@ Boxes *create_boxes_from_line_stream(LineStream *line_stream)
 
     if (sscanf(
             line_stream_next(line_stream),
+#ifdef X64_BUILD
```

Comment:
Does `X64_BUILD` make exist on all of the platforms and compilers?

Reply:
no actually no, i've defined it myself in the compilation process with the -D equivalent for the vs compiler cl.exe.
maybe i can use some define marcos from SDL2, if there's any?